spec/models/department_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Department, type: :model do
+  context 'creating department' do
+    let(:department) { build :department }
+    let(:department2) { build :department, department_name: department.department_name }
+
+    it 'is valid' do
+      expect(department).to be_valid
+    end
+
+    it 'is invalid' do
+      expect(department2[:department_name]).to eq(department[:department_name])
```

Comment:
Question: if department name is eq then how it is invalid? may be we can check some other logic here.